app/assets/json/additional-resources.json

```diff
@@ -0,0 +1,70 @@
+{
+    "categories": [
+        {
+            "title": "assistance.additionalResources.sfHousingResources.title",
+            "subtitle": "assistance.additionalResources.sfHousingResources.subtitle",
+            "resources": [
+                {
+                    "title": "assistance.additionalResources.sfHousingResources.fcfs.title",
+                    "agency": "assistance.additionalResources.sfHousingResources.fcfs.agency",
+                    "description": "assistance.additionalResources.sfHousingResources.fcfs.description",
+                    "externalUrl": "https://sf.gov/reports/january-2023/first-come-first-served-bmr-listings"
```

Comment:
This URL and the next two are out-of-date, but the website redirects to the most updated pages, so it should be fine.